create-oci.sh

```diff
@@ -109,7 +109,7 @@ if [ ${#artifacts[@]} != 0 ]; then
     select-oci-auth.sh "$repo" > "$authfile"
 
     pushd "${archive_dir}" > /dev/null
-    retry.sh oras push "${oras_opts[@]}" --registry-config "$authfile" "${store}" "${artifacts[@]}"
+    /usr/bin/retry oras push "${oras_opts[@]}" --registry-config "$authfile" "${store}" "${artifacts[@]}"
```

Comment:
Let's just use `retry` here. `/usr/bin` or `/usr/local/bin` should already be in PATH.